source/precalculus/source/08-TE/01.ptx

```diff
@@ -12,6 +12,1187 @@
+      </task>
+    </activity>
+
+    <definition xml:id="def-power-reducing-formulas">
+      <p>The trigonometric power reduction formulas allow us to rewrite expressions involving trigonometric terms with trigonometric terms of smaller powers.
+      </p>
+      <p>
+        The <term>Power Reduction Formulas</term> are:
+        <ul>
+          <li>
+            <p>
+              <m>\sin^2\theta=\frac{1-\cos2\theta}{2}</m>
+            </p>
+          </li>
+          <li>
+            <p>
+              <m>\cos^2\theta=\frac{1+\cos2\theta}{2}</m>
+            </p>
+          </li>
+          <li>
+            <p>
+              <m>\tan^2\theta=\frac{1-\cos2\theta}{1+\cos\theta}</m>
+            </p>
+          </li>
+        </ul>
+      </p>
+    </definition>
+
+    <activity xml:id="power-reducing-example">
+
+      <introduction>
+        <p>
+          Rewrite <m>\sin^4\beta</m> as an expression without powers greater than one using.
+        </p>
+      </introduction> 
+
+      <task>
+        <statement>
+          <p>
+            How can we rewrite <m>\sin^4\beta</m> as a function squared?
+          </p>
+        </statement>
+        <answer>
+          <p>
+            <m>\sin^4\beta=\left(\sin^2\beta\right)^2</m>
+          </p>
+        </answer>
+      </task>
+
+      <task>
+        <statement>
+          <p>
+            Rewrite <m>\sin^2\beta</m> using <xref ref="def-power-reducing-formulas"/>.
+          </p>
+        </statement>
+        <answer>
+          <p>
+            <m>\sin^4\beta=\left(\frac{1-\cos2\beta}{2}\right)^2</m>
+          </p>
+        </answer>
+      </task>
+
+      <task>
+        <statement>
+          <p>
+            Square the right-hand side of the equation you found in part (b).
+          </p>
+        </statement>
+        <answer>
+          <p>
+            <m>\sin^4\beta=\frac{1-2\cos2\beta+\cos^22\beta}{4}</m>
+          </p>
+        </answer>
+      </task>
+
+      <task>
+        <statement>
+          <p>
+            Notice that you still have a cosine function being squared in your equation in part (c). Substitute the <m>\cos^2\beta</m> using the power reduction formula in <xref ref="def-power-reducing-formulas"/> and then simplify
+          </p>
+        </statement>
+        <answer>
+          <p>
+            <m>\frac{2-4\cos\beta+1+\cos4\beta}{8}</m> or <m>\frac{3-4\cos2\beta+\cos4\beta}{8}</m>
+          </p>
+        </answer> 
+      </task>
+    </activity>
+
+    <activity xml:id="deriving-half-angle-formulas">
+      <introduction>
+        <p>
+          The next set of identities is the set of half-angle formulas, which can be derived from the reduction formulas. Let's derive the half-angle formula for <m>\sin\frac{\theta}{2}</m>.
+        </p>
+      </introduction>
+
+      <task>
+        <statement>
+          <p>
+            Recall the power reduction formula: <m>\sin^2\theta=\frac{1-\cos2\theta}{2}</m>. Replace <m>\theta</m> with <m>\frac{\theta}{2}</m>.
```

Comment:
Would this be easier to have students set $\theta = \frac{\alpha}{2}$ so that $\theta$ isn't overloaded?  If not, then I think the following definition[sic] should also use $\theta$ to match the activity.